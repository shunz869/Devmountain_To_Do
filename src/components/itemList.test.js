import React from 'react';
import ItemList from './itemList';
import renderer from 'react-test-renderer';

test('ItemList renders correctly', ()=>{
  const component = renderer.create(<ItemList items={[{
    id:1,
    title:'abc'
  }]} />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})