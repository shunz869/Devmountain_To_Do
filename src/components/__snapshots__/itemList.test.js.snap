// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemList renders correctly 1`] = `
<div
  className="row"
  style={
    Object {
      "marginTop": 50,
    }
  }
>
  <div
    className="col-12"
  >
    <ul
      className="list-group"
    >
      <li
        className="list-group-item d-flex justify-content-between align-items-center"
      >
        <span
          onClick={[Function]}
        >
          abc
        </span>
        <div
          className="btn-group"
        >
          <button
            className="btn btn-secondary btn-sm"
            onClick={[Function]}
            style={
              Object {
                "marginRight": 10,
              }
            }
          >
            Complete
          </button>
          <button
            className="btn btn-secondary btn-sm"
            onClick={[Function]}
          >
            Delete
          </button>
        </div>
      </li>
    </ul>
  </div>
</div>
`;
